src/components/diagrams/diagram-pane.js

```diff
@@ -620,6 +637,26 @@ export function DiagramPane({
         ]
     );
 
+    const handleWarning = (id, svgType) => {
+        // Add the id of the diagramView and the warning to display based on svg type
+        const isVoltageLevel = DiagramType.VOLTAGE_LEVEL === svgType;
+        const message = isVoltageLevel
+            ? 'VoltageLevelNotFound'
+            : 'SubstationNotFound';
+        setWarnings(
+            (prev) => new Map(prev.set(id, disabled ? 'InvalidNode' : message))
+        );
+    };
+
+    const hadleWarningToDisplay = (diagramView) => {
+        // First, check if the node is built(the highest priority) then do the warning checks..
+        if (disabled) return 'InvalidNode';
+        if (diagramView.isSvgNotFound) return diagramView?.messageNotFound;
```

Comment:
Instead of testing for isSvgNotFound, we could instead return diagramView.messageNotFound if it is defined.

Reply:
done